modules/local/format_pplacetax.nf

```diff
@@ -0,0 +1,96 @@
+process FORMAT_PPLACETAX {
+    tag "${tax.baseName}"
+    label 'process_high'
+
+    conda "bioconda::bioconductor-dada2=1.22.0 conda-forge::r-digest=0.6.30"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bioconductor-dada2:1.22.0--r41h399db7b_0' :
+        'quay.io/biocontainers/bioconductor-dada2:1.22.0--r41h399db7b_0' }"
```

Comment:
Is dada2 needed for this module?

Reply:
No, just base R. But the DADA2 container is used in any case in the pipeline, so it should save bandwidth and time to reuse it.